pdi/CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ and this project adheres to
 #### Removed
 
 #### Fixed
+* Aligned memory allocation using `operator new`, fixes
+  [#550](https://github.com/pdidev/pdi/issues/550) 
+* Fix compile issue with macOS where `std::numeric_limits<uint64_t>::min()` is interpreted as `unsigned long long`
```

Comment:
Maybe we could open a "mother issue" about "compatibility with MacOS", and only reference this one here. I guess the typical user doesn't really care about the technical details that had to be handled for this.